docs/output.md

```diff
@@ -98,12 +114,21 @@ The `adapterqc` stage checks for the presence and correctness of the pixel bindi
 
 </details>
 
-The `pixelator single-cell demux` command assigns a marker (barcode) to each read. It also generates QC report in
-JSON format. It saves processed reads (one per antibody) as well as discarded reads with no match to the
-given barcodes/antibodies.
-
 ### Duplicate removal and error correction
 
+This step uses the `pixelator single-cell collapse` command.
+
+The `collapse` command removes duplicate reads and performs error correction.
```

Comment:
```suggestion
The `collapse` command removes duplicate reads and performs error correction, resulting in the molecules that have been sampled in the experiment.
```

Reply:
I find that this additional does not addition does not add much information and makes it even a bit less clear.
Can we leave it as it was?

Reply:
I think my issue here is that we introduce `collapse` but we don't mention that the main purpose of this step is to discover molecules, not to remove duplicate reads and perform error molecules. Therefore we use the UPI and UMI (in the sentence after).